### ISSUE TITLE
Incorrect warning message for COMPLEX_BOOLEAN_EXPRESSION rule

### ISSUE THREAD

**Issue by akuleshov7** [cqfn/diKTat#1040]
## Describe the bug
The following boolean condition triggers a warning with incorrect text:

```
        return if ((node is TomlKeyValueSimple) || (node is TomlKeyValueSimple)) {
            decodeValue().toString().toLowerCase() != "null"
        } else {
            // all other technical nodes (like tables) should not contain null values
            // so we will mark them with a not null mark
            true
        }
```

## Expected behavior
```
 [COMPLEX_BOOLEAN_EXPRESSION] too complex boolean expression, that can be simplified: (node
 is TomlKeyValueSimple) 
```
## Observed behavior
```
 [COMPLEX_BOOLEAN_EXPRESSION] too complex boolean expression, that can be simplified: (node
 is TomlKeyValueSimple) || (node is TomlKeyValueSimple)
```
## Environment information
* diktat version: `1.0.0-rc.2`

**Comment by kgevorkyan** [comment]
Actually, that is not a bug: diktat tell us **which node could be** simplified, and **not how it will be** simplified
I will expand a warning message.

However, in fix mode, we've got an NPE for this example, which is a real problem

```bash
 2021-09-13 16:35:24 Internal error has occurred in rule [boolean-expressions-rule]. Please make an issue on this bug at https://github.com/cqfn/diKTat/. 
                       As a workaround you can disable these inspections in yml config: <[COMPLEX_BOOLEAN_EXPRESSION]>.
                       Root cause of the problem is in [D:\diKTat\diktat-rules\target\test-classes\test\paragraph3\boolean_expressions\SameExpressionsInConditionTest.kt] file.
org.cqfn.diktat.ruleset.utils.KotlinParseException: Text is not valid: []
	at org.cqfn.diktat.ruleset.utils.KotlinParser.createNode(KotlinParser.kt:75)
	at org.cqfn.diktat.ruleset.utils.KotlinParser.createNode$default(KotlinParser.kt:75)
	at org.cqfn.diktat.ruleset.rules.chapter3.BooleanExpressionsRule.fixBooleanExpression(BooleanExpressionsRule.kt:167)
	at org.cqfn.diktat.ruleset.rules.chapter3.BooleanExpressionsRule.access$fixBooleanExpression(BooleanExpressionsRule.kt:34)
	at org.cqfn.diktat.ruleset.rules.chapter3.BooleanExpressionsRule$checkBooleanExpression$1.invoke(BooleanExpressionsRule.kt:72)
	at org.cqfn.diktat.ruleset.rules.chapter3.BooleanExpressionsRule$checkBooleanExpression$1.invoke(BooleanExpressionsRule.kt:71)
	at org.cqfn.diktat.ruleset.constants.Warnings.warnAndFix(Warnings.kt:260)
```
